main.js

```diff
@@ -0,0 +1,258 @@
+//Week3 get Hours
+const d = new Date();
+const currentHour = d.getHours();
+console.log(currentHour);
+
+const openingHour = 9;
+const closingHour = 17;
+
+const marketAvailability = document.getElementById('marketAvailability')
+if(currentHour > openingHour && currentHour < closingHour){
```

Comment:
if the opening hours is at 9 and closing at 17 the condition should look like this `currentHour >= openingHour && currentHour <= closingHour`

also, this will be called only once when the user opens the app, but you want to inform him for the whole time the app is open, for this purpose you can use `setInterval` method with timer set to 1 min for example :)